api/src/config.ts

```diff
@@ -0,0 +1,16 @@
+const getEnvString = (name: string): string => process.env[name] || '';
+const getEnvNumber = (name: string): number => parseInt(getEnvString(name), 10);
+const getEnvBoolean = (name: string): boolean => getEnvString(name) === 'true';
```

Comment:
this is brittle and likely to cause hard to debug issues. E.g. `True` or even ` true` (with a leading space) will cause issues that aren't obvious. I'd make this a full function with an `assert value === 'false'` if the `=== true` branch false.